generate_wordstr_box.py

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env python3
 
 import argparse
-import bidi.algorithm
 import io
 import unicodedata
+
```

Comment:
Some Python coding standards and their related formatters sort the import statements alphabetically. That's why I did likewise. What was the motivation for moving the import statement here to a different line?

Reply:
Sorry, I should have looked at your commit message first. So pylint groups import statements, starting with standard imports, then third party imports. So that's fine. `isort` also implements that order.